options.go

```diff
@@ -13,12 +13,23 @@ type Options struct {
 	// Print step definitions found and exit
 	ShowStepDefinitions bool
 
-	// Run scenarios in random order.
+	// Randomize causes scenarios to be run in random order.
 	//
-	// This is especially helpful for detecting situations
-	// where you have state leaking between scenarios, which
-	// can cause flickering or fragile tests.
-	RandomOrder bool
+	// Randomizing scenario order is especially helpful for detecting
+	// situations where you have state leaking between scenarios, which can
+	// cause flickering or fragile tests.
+	Randomize bool
```

Comment:
this option becomes unnecessary, when seed is used it is inferred to be true. why not just changing it to one option **Randomize int64** which is a seed. default value `0` means do not randomize. A single option for single task, easier to understand from option list.